articles/libraries/lock-android/v2/passwordless.md

```diff
@@ -5,9 +5,7 @@ description: Guide on implementing Passwordless authentication with Lock for And
 ---
 # Lock Android: Passwordless
 
-::: warning
-Passwordless on native platforms is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case. See [Client Grant Types](/clients/client-grant-types) for more information. Alternatively, you can use Lock Passwordless on Auth0's [Hosted Login Page](/hosted-pages/login).
-:::
+<%= include('../../../_includes/_native_passwordless_warning') %>
```

Comment:
The v1 doc still has the old warning, checking if this was done deliberately

https://auth0-docs-content-pr-5633.herokuapp.com/docs/libraries/lock-android/v1/passwordless

Reply:
Good catch